src/services/WorkspaceFlowRunsApi.ts

```diff
@@ -26,10 +28,20 @@ export class WorkspaceFlowRunsApi extends WorkspaceApi implements IWorkspaceFlow
 
   protected routePrefix = '/flow_runs'
 
-  public async getFlowRun(id: string): Promise<FlowRun> {
-    const { data } = await this.get<FlowRunResponse>(`/${id}`)
+  private readonly batcher = new Batcher<string, FlowRun>(async ids => {
+    const flowRuns = await this.getFlowRuns({
```

Comment:
technically `getFlowRuns` could be making its _own_ batch call under the hood 😮

Reply:
Technically yes. But that would be difficult because of filters. I'm not sure how you would decide if two filters could be combined into one request and then the callback for that Hatcher would be really gross lol.